themes/ddbasic/template.node.php

```diff
@@ -0,0 +1,369 @@
+  }
+
+  switch ($variables['view_mode']) {
+    case 'teaser':
+      // Add class if image.
+      if (!empty($variables['field_ding_event_list_image'])) {
+        $variables['classes_array'][] = 'has-image';
+      }
+      // Create image url.
+      $uri = empty($variables['field_ding_event_list_image'][0]['uri']) ?
+        "" : $variables['field_ding_event_list_image'][0]['uri'];
+
+      if (!empty($uri)) {
+        $variables['event_background_image'] = image_style_url('ding_panorama_list_large', $uri);
+      }
+
+      $variables['image_title'] = empty($variables['field_ding_event_list_image'][0]['title']) ?
+        "" : 'title="' . $variables['field_ding_event_list_image'][0]['title'] . '"';
+
+      // Date.
+      if (!empty($date)) {
+        $start_date = strtotime($date[0]['value']);
+        $end_date = strtotime($date[0]['value2']);
+
+        $variables['event_date'] = format_date($start_date, 'ding_date_only_version2');
+        $event_time_view_settings = array(
+          'label' => 'hidden',
+          'type' => 'date_default',
+          'settings' => array(
+            'format_type' => 'ding_time_only',
+            'fromto' => 'value',
+          ),
+        );
+
+        // If start and end date days are equal.
+        if (date('Ymd', $start_date) !== date('Ymd', $end_date)) {
+          $variables['event_date'] .= ' - ' . format_date($end_date, 'ding_date_only_version2');
+        }
+        // If start and end date days and time are not equal.
+        if ($start_date !== $end_date) {
+          $event_time_view_settings['settings']['fromto'] = 'both';
+        }
+
+        $event_time_ra = field_view_field('node', $variables['node'], 'field_ding_event_date', $event_time_view_settings);
+        $variables['event_time'] = $event_time_ra[0]['#markup'];
+      }
+
+      break;
+
+    case 'full':
+      if (!empty($date)) {
+        array_push($variables['classes_array'], 'node-full');
+
+        // Add event time to variables. A render array is created based on the
+        // date format "time_only".
+        $event_time_ra = field_view_field('node', $variables['node'], 'field_ding_event_date', array(
+          'label' => 'hidden',
+          'type' => 'date_default',
+          'settings' => array(
+            'format_type' => 'ding_time_only',
+            'fromto' => 'both',
+          ),
+        ));
+        $variables['event_time'] = $event_time_ra[0]['#markup'];
+
+        // Make social-share button.
+        $share = '
+        <div class="social-share-container">
+          <div class="inner">
+            <div class="label">' . t('Share this event') . '</div>
+            <div class="share-buttons">
+              <a href="#" class="fb-share">Facebook</a>
+              <a href="#" class="twitter-share">Twitter</a>
+              <a href="#" class="mail-share">E-mail</a>
+            </div>
+          </div>
+        </div>
+      ';
+        $variables['share_button'] = $share;
```

Comment:
As above. It is a good thing that the social sharing elements are no longer presented as a field.